src/python/pants/backend/codegen/protobuf/python/python_protobuf_subsystem.py

```diff
@@ -82,6 +102,17 @@ class PythonProtobufMypyPlugin(PythonToolRequirementsBase):
     default_lockfile_resource = ("pants.backend.codegen.protobuf.python", "mypy_protobuf.lock")
 
 
+class PythonProtobufGrpclibPlugin(PythonToolRequirementsBase):
+    options_scope = "grpclib"
+    help = "Configuration of the grpclib plugin."
+
+    default_requirements = ["grpclib[protobuf]>=0.4"]
```

Comment:
We like to upper bound these, so that regenerating the lockfile doesn't suddenly give you some incompatible version. So this should probably be `grpclib[protobuf]>=0.4,<1`